src/Movies/Movies.js

```diff
@@ -8,13 +8,14 @@ class Movies extends Component {
     super()
     this.state = {
       movies: [],
+      movieResults: [],
       searchInput: "",
       error: null
     }
   }
 
   handleChange = (event) => {
-    this.setState({ searchInput: event.target.value })
+    this.setState({ searchInput: event.target.value, moviesResults: this.state.movies })
```

Comment:
Good idea adding a new key to handle change of state of the movies so we don't override all the movies available